### DISCUSSION TITLE
Someone explain to me why Mongo with typeorm is used monogdb is nosql database?

### DISCUSSION THREAD

**Discussion by TechVelvet** [brocoders/nestjs-boilerplate#1668] [Q&A] [answered]
Someone explain to me why Mongo with typeorm is used monogdb is nosql database?
Using TypeORM with MongoDB might seem unusual since MongoDB is a NoSQL database, and TypeORM is primarily an ORM designed for relational databases.

**Comment by Shchepotin** [comment, marked as answer]
@TechVelvet NestJS Boilerplate is use Mongoose for MongoDB. About "TypeORM with MongoDB" you can ask here: https://github.com/typeorm/typeorm

**Reply by amainooti** [reply]
Yeah it supports mongodb even tho it's unstructured
